### PR TITLE
devtools install_github does not work

### PR DESCRIPTION
**Pull request by romainx**
`install_github` is an handy tool to install R packages directly from GitHub repositories (see [Putting your R package on GitHub](http://kbroman.org/pkg_primer/pages/github.html) for more information). In the `r-notebook` this method does not working.
```bash
$ R
> devtools::install_github("hadley/babynames")
# Downloading GitHub repo hadley/babynames@master
# from URL https://api.github.com/repos/hadley/babynames/zipball/master
# Installation failed: error in running command
```

To fix it I have bumped the version of devtools to `2.0*`, but it was still not OK. A problem related to the `tar` utility remains.
When telling R to give the content of the `TAR` environment variable this is its answer.
```bash
$ R
> Sys.getenv("TAR")
[1] "/bin/gtar"
```
Since `/bin/gtar` does not exist on the platform, it does not work.
This problem is known:
- [`install_github` fails in conda environment on Linux · Issue #4 · conda-forge/r-devtools-feedstock · GitHub](https://github.com/conda-forge/r-devtools-feedstock/issues/4)
 - [Installation failed: error in running command · Issue #1722 · r-lib/devtools · GitHub](https://github.com/r-lib/devtools/issues/1722)
 - [release function error · Issue #379 · r-lib/devtools · GitHub](https://github.com/r-lib/devtools/issues/379) 

To fix it two pragmatic approaches are
- To create a symlink `ln -s /bin/tar /bin/gtar`
- To fix the environment variable in the user  `.Renviron` file

I’ve chosen the latter.

```Dockerfile
RUN echo "TAR=/bin/tar" > $HOME/.Renviron
```

### CODE REVIEW

**Comment by romainx** [comment]
The build error is not related to this change that impact ` r-notebook` but to `pyspark-notebook`.

```
The command '/bin/sh -c apt-get -y update &&     apt-get install --no-install-recommends -y openjdk-8-jre-headless ca-certificates-java &&     rm -rf /var/lib/apt/lists/*' returned a non-zero code: 100
make: *** [build/pyspark-notebook] Error 100
```

File: r-notebook/Dockerfile [hunk 1]
```diff
@@ -46,3 +46,6 @@ RUN conda install --quiet --yes \
     'r-hexbin=1.27*' && \
     conda clean -tipsy && \
     fix-permissions $CONDA_DIR
+
+# Fix for devtools https://github.com/conda-forge/r-devtools-feedstock/issues/4
+RUN echo "TAR=/bin/tar" > $HOME/.Renviron
```
Comment (parente) [inline r-notebook/Dockerfile]: Users of these images often mount a host fodler as the home directory. In that case, the `.Renviron` file will be missing. Is there a another location R looks for configuration  (e.g., somewhere in `/etc`) where we can put this file instead?

**Comment by parente** [comment]
@romainx Thanks for the PR. Just the one question.

**Comment by romainx** [comment]
@parente No problem.

I agree with you it's not the best solution. I will change the pull request to implement the second solution.
`ln -s /bin/tar /bin/gtar`

**Comment by parente** [comment]
It looks reasonable to me to do that symlinking in the r-notebook image. Maybe it'll cause a problem for someone installing the real `gtar` later, but let's create the problem and find out.

**Pull request merged**